modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/WriteActionRequest.java

```diff
@@ -19,15 +19,23 @@
 
 import org.apache.ignite.internal.raft.WriteCommand;
 import org.apache.ignite.network.annotations.Transferable;
+import org.apache.ignite.network.annotations.Transient;
 import org.apache.ignite.raft.jraft.RaftMessageGroup.RpcActionMessageGroup;
+import org.jetbrains.annotations.Nullable;
 
 /**
  * Submit a  write action to a replication group.
  */
 @Transferable(RpcActionMessageGroup.WRITE_ACTION_REQUEST)
 public interface WriteActionRequest extends ActionRequest {
     /**
-     * Returns an action's command.
+     * @return Serialized action's command. Specific serialization format may differ from group to group.
      */
-    WriteCommand command();
+    byte[] command();
+
+    /**
+     * @return Original non-serialized command, if available. {@code null} if not.
```

Comment:
Please expand the description and describe why this field is and why it should be used and when it will be `null`.